Tests/TuistGeneratorTests/Generator/TargetGeneratorTests.swift

```diff
@@ -107,6 +107,48 @@ final class TargetGeneratorTests: XCTestCase {
         ])
     }
 
+    func test_generateTarget_actions() throws {
+        /// Given
```

Comment:
I think we mostly use `//` (not `///`) for given/when/than as it's not a documentation.